verify_user.js

```diff
@@ -22,8 +22,9 @@ function VerifyToken(req, res, next) {
     'Access-Control-Allow-Headers',
     'Origin, X-Requested-With, Content-Type, Accept'
   )
-  res.header("Access-Control-Allow-Origin", "*");
+  res.header("Access-Control-Allow-Origin", "http://www.digitalbraininjury.com/");
```

Comment:
can the domain made into a variable and passed in from configuration file? Soon we will be moving to nsfcareer.io